src/logging.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
This is part new work and part copypasta from `main.js`. The new work is to configure winston itself and to add our formatters.